File: tests/py/example2.py
# Fonction pour transformer le texte
def TransformText(text):
    text = text.upper()  # Convertir en majuscules
    prefix = "Prefix: "
    return AddPrefix(modifiedText, prefix)

# Fonction pour ajouter un préfixe
def AddPrefix(text, prefix):
    return prefix + text

# Fonction de test
def test():
    inputText = "Hello, World!"
    result = TransformText(inputText)
    print(result)

# Fonction principale
if __name__ == "__main__":
    test()

File: tests/js/example2.js
// Fonction pour transformer le texte
function TransformText(text) {
    let text = text.toUpperCase();  // Conversion en majuscules
    let prefix = "Prefix: ";
    let finalText = AddPrefix(modifiedText, prefix);  // Appel de la fonction AddPrefix
    return finalText;
}

// Fonction pour ajouter un préfixe au texte
function AddPrefix(text, prefix) {
    return prefix + text;
}

// Fonction de test
function test() {
    let inputText = "Hello, World!";
    let result = TransformText(inputText);
    console.log(result);
}

// Fonction principale
function main() {
    test();
}

// Appel de la fonction principale
main();